deploy/charts/kiali-operator/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: kiali-operator
+description: Kiali is an open source project for service mesh observability, refer to https://www.kiali.io for details.
+version: 1.0.0
+appVersion: 1.0.0
```

Comment:
don't we want the appVersion of Kiali here? 1.0.0 implies the (very old) Kiali. I think we'd want some kind of ${xyz} environment variable replacement here so we can build the charts for a specific version of Kiali.

Reply:
During the chart artifact creation I set the appVersion there, I was thinking the less which needs to be updated the better. This way the artifact is always inline with the released version but nobody needs to come in here and make a change.

Reply:
Ah.. OK. if its changed automagically on chart creation, you can ignore me.